### PR TITLE
[FIX] Minor bugfixes

### PR DESCRIPTION
**Pull request by gdgellatly**
Printer Tray isn't an app
Base Report To printer ir.actions.report.xml view xpath error and missing page name
Put to before security as security tab doesn't do much and if this module is installed it should be the hero page

### CODE REVIEW

**Review by Garamotte — APPROVED** [review]

**Comment by gdgellatly** [comment]
I have no idea why Travis failed - its literally a manifest and view change but its stalled the whole build and says cups server not reachable.

**Comment by Garamotte** [comment]
FYI, I fixed the timeout issue in #95

**Comment by Garamotte** [comment]
@gdgellatly #95 is merged, you can rebase on 10.0 to fix Travis.

**Comment by gdgellatly** [comment]
@sylvain-garancher rebased

**Comment by Garamotte** [comment]
@gdgellatly Could you rebase (again) on 10.0, now that the wkhtmltopdf issue has been fixed (https://github.com/OCA/report-print-send/pull/107).

**Comment by gdgellatly** [comment]
@sylvain-garancher is rebase really necessary, it should merge fine.  Its just I got myself in a mess with this branch as I was doing v11 port work and did it on wrong branch.

If it is needed, I'll just close this PR and redo from scratch as its only 3 lines.

**Review by lasley — APPROVED** [review]
@gdgellatly - We do need a rebase though to make sure CI goes 🍏  before merge. I can perform this if you'd like (make sure that you've enabled the "Allow edits from maintainers" option in this PR)

**Comment by gdgellatly** [comment]
@lasley its enabled, but I'll give it a go, call it a learning experience.  Pretty sure I can just check out that remote as a new branch and rebase maybe.

**Comment by gdgellatly** [comment]
@lasley all green now.  Sorry for the rebase moan, I get git in a mess a lot of the time through not thinking and can't see a way out.  Decided to call it spagitti.

**Comment by lasley** [comment]
Git's a bitch sometimes, that's for sure. Merging

**Pull request merged**
